src/lib/litegraph/src/LGraphCanvas.ts

```diff
@@ -2851,6 +2851,12 @@ export class LGraphCanvas
 
     this.dirty_canvas = true
   }
+  passPointerEvent(pointer: CanvasPointer, node: LGraphNode): boolean {
```

Comment:
If it's an intentional mutation, I usually like to note that in the function name.
Something like `augmentPointerDragEvents` maybe?